key.core/src/main/java/de/uka/ilkd/key/speclang/njml/JmlTermFactory.java

```diff
@@ -920,14 +920,14 @@ public Term createLocSet(ImmutableList<SLExpression> exprList) {
                         singletons = singletons.append(t);
                     } else if (t.op() instanceof ProgramVariable) {
                         // this case may happen with local variables
-                        exc.addIgnoreWarning("local variable in assignable clause");
+                        exc.addIgnoreWarning("local variable in modifiable clause");
                         LOGGER.debug("Can't create a locset from local variable " + t + ".\n"
```

Comment:
```suggestion
                        LOGGER.debug("Cannot create a locset from local variable " + t + ".\n"
```
Just to be consistent with the error messages ;-)

Reply:
Would we really make modifyable the new default keyword?

Reply:
I changed the two error messages accordingly. As mentioned, this pull request does not infer or generate any specification. Hence, other than in the error messages, the user is not confronted with any new keywords, if they did not specify them themselves in the (user-given) program specification.